readme.md

```diff
@@ -267,6 +267,7 @@
 	- [NLP with Ruby](https://github.com/arbox/nlp-with-ruby)
 - [Linguistics](https://github.com/theimpossibleastronaut/awesome-linguistics)
 - [Cryptography](https://github.com/sobolevn/awesome-cryptography)
+- [Crypto Papers](https://github.com/pFarb/awesome-crypto-papers) - focusing on theoretical background (papers, books) for using cryptographic algorithms for non-cryptographers.
```

Comment:
This should be a sub-item of the `Cryptography` entry (See other examples for that above).

`focusing` => `Focusing`

And the description needs to be shorter.